cms/static/sass/course-unit-mfe-iframe-bundle.scss

```diff
@@ -39,67 +44,105 @@ html {
 
       .actions-list .action-item .action-button {
         border-radius: 4px;
+        display: inline-flex;
+        align-items: center;
+        gap: 0.6rem;
+        padding: 0.3rem 1rem;
```

Comment:
[question]: We don't use rems in this file. Should we be consistent or do we want to use rems everywhere?

Reply:
Replaced with $baseline, thanks

Reply:
Resolved